javascript/javascript1/week3/smartphoneapp.js

```diff
@@ -0,0 +1,80 @@
+const activities = []
+function addActivity(activity, duration){
+    var today = new Date();
+today = today.toLocaleDateString("en-US");
+activities.push({
+        date: today,
+        activity: activity,
+        duration: duration,
+})
+}
+addActivity("Youtube", 30);
+function showStatus(activities){
+    if(activities.length==0){
+        return "Add some activities before calling showStatus"
+    }
+    let totalDuration = 0
+    for(let i=0;i<activities.length;i++)
+    {
+        totalDuration += activities[i].duration
+    }
+    const limit = 120
+    if(totalDuration>=limit){alert('You have reached your limit, no more smartphoning for you!')}
+return `You have added ${activities.length} activities. You have ${totalDuration}  min. of usage`
+}
+showStatus(activities); // will log out this "You have added 3 activities. They amount to 78 min. of usage"
```

Comment:
Have you checked if you can see the log message on your console? Hint: you are returning the string but not logging into console.

Reply:
done